scripts/travis/start_elasticdl_job.sh

```diff
@@ -0,0 +1,56 @@
+JOB_TYPE=$1
```

Comment:
The word "start" means to start the job but not waiting for the completion. However, this script does wait for the completion of the job.  So it should be renamed `run_job.sh`.